crates/node/src/chain_spec.rs

```diff
@@ -91,19 +92,21 @@ pub fn development_config(enable_manual_seal: Option<bool>, madara_path: PathBuf
 // helper to print development accounts info
 // accounts with addresses 0x1 and 0x4 are NO VALIDATE accounts (don't require PK)
 // accounts with addresses 0x2 and 0x3 have the same PK
-pub fn print_development_accounts(genesis_loader: GenesisLoader) {
-    let genesis_config: madara_runtime::pallet_starknet::GenesisConfig<Runtime> = genesis_loader.into();
-    let no_validate_account_address = genesis_config.contracts[0].0.0.to_string();
-    let argent_account_address = genesis_config.contracts[1].0.0.to_string();
-    let oz_account_address = genesis_config.contracts[2].0.0.to_string();
-    let cairo_1_no_validate_account_address = genesis_config.contracts[3].0.0.to_string();
-
-    const ARGENT_PK: &str = "0x00c1cf1490de1352865301bb8705143f3ef938f97fdf892f1090dcb5ac7bcd1d";
+pub fn print_development_accounts(genesis_loader: &GenesisLoader) {
+    let no_validate_account_address = genesis_loader.contracts[0].0.0;
+    let argent_account_address = genesis_loader.contracts[1].0.0;
+    let oz_account_address = genesis_loader.contracts[2].0.0;
+    let cairo_1_no_validate_account_address = genesis_loader.contracts[3].0.0;
+
+    let argent_pk: HexFelt =
+        Felt252Wrapper::from_hex_be("0x00c1cf1490de1352865301bb8705143f3ef938f97fdf892f1090dcb5ac7bcd1d")
+            .unwrap()
+            .into();
     log::info!("🧪 Using the following development accounts:");
-    log::info!("🧪 NO VALIDATE with address: 0x{} and no pk", no_validate_account_address);
-    log::info!("🧪 ARGENT with address: 0x{} and pk: {}", argent_account_address, ARGENT_PK);
-    log::info!("🧪 OZ with address: 0x{} and pk: {}", oz_account_address, ARGENT_PK);
-    log::info!("🧪 CAIRO 1 with address: 0x{} and no pk", cairo_1_no_validate_account_address);
+    log::info!("{}", format!("🧪 NO VALIDATE with address: {no_validate_account_address:#x} and no pk"));
```

Comment:
Can't you get rid of the format? Since you already have a `"{}"` in the `log::info!`